### PR TITLE
Added camel case aliases for all public methods and exported functions

### PR DESCRIPTION
**Pull request by sampeka**
As per #3. 

Also included some test coverage for the added helpers.

### CODE REVIEW

**Comment by sampeka** [comment]
It's failing in node 0.12 because `string.startsWith` isn't there. Is there a way we can just configure babel to handle that? Or would you rather I just rewrite it slightly? Not a big deal either way.

**Comment by halt-hammerzeit** [comment]
Hello.
I looked through your Pull Request and I understand only 
```
aliasCamelCaseAttrs(webpack_isomorphic_tools.prototype)
```
and
```
function aliasCamelCaseAttrs() { ... }
function camelCase() { ... }
```
I don't see the purpose of all the other code (except tests)

**Comment by sampeka** [comment]
The rest just makes camel case aliases for all exported functions and adds them to `module.exports`. It's not totally necessary to have `helpers` and the other modules also export camel case, but it makes it more consistent if somebody does for whatever reason decide to use any of them in their own code. If you disagree though I'll strip the PR back to just `index.js`

File: source/common.js [hunk 1]
```diff
@@ -278,3 +278,5 @@ export function uniform_path(asset_path)
 
 	return asset_path
 }
+
+exportAllCamelCase(module);
```
Comment (halt-hammerzeit) [inline source/common.js]: is it required for any use case?

File: source/plugin/notify stats.js [hunk 2]
```diff
@@ -51,4 +52,6 @@ export default function notify_stats(stats, json, verbose)
 
 		first_run = false
 	}
-}
\ No newline at end of file
+}
+
+exportAllCamelCase(module)
```
Comment (halt-hammerzeit) [inline source/plugin/notify stats.js]: is it required for any use case?

Comment (sampeka) [inline source/plugin/notify stats.js]: I guess not ;) I'll remove the module-level stuff

**Comment by sampeka** [comment]
Done, removed the module-level stuff. Also added an extra test to make sure the camel case attributes have been properly added to the isomorpher prototype. If you're happy, let me know and I'll squash the commits.

**Comment by halt-hammerzeit** [comment]
@sampeka ok, that looks good to go.
I'll accept the PR after you squash it

**Comment by sampeka** [comment]
Squashed, also just realised that people use `plugin.js` directly as well. Not all of the methods had camel case variants, so did the same there + added some light coverage.

**Comment by halt-hammerzeit** [comment]
Ok, your contribution accepted.
i'll go release a new version.

**Comment by halt-hammerzeit** [comment]
done
`webpack-isomorphic-tools@2.2.44`

**Comment by halt-hammerzeit** [comment]
FYI
I'm posting this in every issue and PR to notify whoever may be interested:
today I've released an alternative helper library called [`universal-webpack`](https://github.com/halt-hammerzeit/universal-webpack).
It takes a different approach than `webpack-ismorphic-tools` and instead of hacking Node.js `require()` calls it just compiles all code with `target: 'node'` webpack configuration option.
As a result, all Webpack plugins and features are supported.
If you think you might need that here's an example project:
https://github.com/halt-hammerzeit/webpack-react-redux-isomorphic-render-example

**Pull request merged**